edx_adapt/api/resources/tutor_resources.py

```diff
@@ -59,26 +62,29 @@ def get(self, course_id, user_id):
                 # then the user knows too much already (ATTENTION_QUESTION_NAME
                 # is only for checking that the user is paying attention,
                 # not knowledge assessment, so skip it)
+                # FIXME(idegtirov) threshold for pre_assessment problems should be changed from int to percentage
                 answers = self.repo.get_all_interactions(course_id, user_id)
-                if sum( [x['correct'] for x in answers if (x['problem']['pretest'] and x['problem']['problem_name'] != UserProblems.ATTENTION_QUESTION_NAME) ] ) > 7:
+                if sum([x['correct'] for x in answers if
+                        (x['problem']['pretest'] and
+                            x['problem']['problem_name'] != UserProblems.ATTENTION_QUESTION_NAME)]) > 7:
                     done_with_course = True
                     nex = None
                 # if answer to ATTENTION_QUESTION_NAME is wrong -
                 # then filter out this user, because they are not paying
                 # attention and simply clicking buttons
                 pretest_done = len(self.repo.get_all_remaining_pretest_problems(course_id, user_id)) == 0
-                # Disable cut off of students who gave no correct answers at pre-assessment for easier debugging
-                # if pretest_done and (sum( [x['correct'] for x in answers if (x['problem']['pretest'] and x['problem']['problem_name'] == UserProblems.ATTENTION_QUESTION_NAME) ] ) < 1):
-                if pretest_done and (sum( [x['correct'] for x in answers if (x['problem']['pretest'] and x['problem']['problem_name'] == UserProblems.ATTENTION_QUESTION_NAME) ] ) < 0):
+                if pretest_done and (
+                    sum([x['correct'] for x in answers if
+                        (x['problem']['pretest'] and
+                            x['problem']['problem_name'] == UserProblems.ATTENTION_QUESTION_NAME)]) < 1):
                     done_with_course = True
                     nex = None
-
-
             except DataException as e:
                 print("--------------------\tDATA EXCEPTION: " + str(e))
                 abort(500, message=str(e))
 
-        return {"next": nex, "current": cur, "done_with_current": done_with_current, "okay": okay, "done_with_course": done_with_course}
+        return {"next": nex, "current": cur, "done_with_current": done_with_current, "okay": okay,
```

Comment:
The same as above.
http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html